obspy/clients/eida/__init__.py

```diff
@@ -10,120 +13,182 @@
+service:
+
+>>> from obspy.clients.eida import Client
+>>> authdata = open("token.asc").read()
+>>> client = Client(authdata=authdata)
+
+For backwards compatibility, username/password authentication is supported as
+well. Different credentials can be specified for each node and they take
+precedence over token authentication at that node.
+
+Add debug=True to see what is going on.
+
+>>> from obspy.clients.eida import Client
+>>> credentials = {"http://service.iris.edu/fdsnws/dataselect/1/queryauth":
+                   ("nobody@iris.edu", "anonymous")}
+>>> authdata = open("token.asc").read()
+>>> client = Client(credentials=credentials, authdata=authdata, debug=True)
+
+(1) :meth:`~obspy.clients.eida.client.Client.get_waveforms()`: The following
+    example illustrates how to request and plot 60 minutes of the ``"LHZ"``
+    channel of EIDA stations starting with ``"A"`` for a seismic event around
+    2010-02-27 07:00 (UTC). Results are returned as a
+    :class:`~obspy.core.stream.Stream` object.
+    See the :meth:`~obspy.clients.eida.client.Client.get_waveforms_bulk()`
+    method for information on how to send multiple requests simultaneously to
+    avoid unnecessary network overhead.
+
+    >>> from obspy import UTCDateTime
+    >>> t = UTCDateTime("2010-02-27T07:00:00.000")
+    >>> st = client.get_waveforms("*", "A*", "", "LHZ", t, t+60*60))
+    >>> st.plot(starttime=t, endtime=t+60*60)  # doctest: +SKIP
+
+    .. plot::
+
+        from obspy import UTCDateTime
+        from obspy.clients.eida import Client
+        client = Client()
+        t = UTCDateTime("2010-02-27T07:00:00.000")
+        st = client.get_waveforms("*", "A*", "", "LHZ", t, t+60*60))
+        st.plot(starttime=t, endtime=t+60*60)
+
+(2) :meth:`~obspy.clients.eida.client.Client.get_events()`: Retrieves event
+    data from the server. Results are returned as a
+    :class:`~obspy.core.event.Catalog` object.
+
+    The event service is not routed, so a node that is running an event service
+    locally must be used.
+
+    >>> client = Client("INGV")
+    >>> starttime = UTCDateTime("2002-01-01")
+    >>> endtime = UTCDateTime("2002-01-02")
+    >>> cat = client.get_events(starttime=starttime, endtime=endtime)
+    >>> print(cat)  # doctest: +NORMALIZE_WHITESPACE
+    4 Event(s) in Catalog:
+    2002-01-01T19:42:34.670000Z | +43.434,  +12.525 | 1.9 Md | manual
+    2002-01-01T10:54:09.040000Z | +43.439,  +12.476 | 2.2 Md | manual
+    2002-01-01T08:21:30.790000Z | +46.071,  +10.630 | 2.3 Md | manual
+    2002-01-01T05:07:31.250000Z | +44.781,   +8.361 | 2.7 Md | manual
+    >>> cat.plot()  # doctest: +SKIP
+
+    .. plot::
+
+        from obspy import UTCDateTime
+        from obspy.clients.eida import Client
+        client = Client("INGV")
+        starttime = UTCDateTime("2002-01-01")
+        endtime = UTCDateTime("2002-01-02")
+        cat = client.get_events(starttime=starttime, endtime=endtime)
+        cat.plot()
```

Comment:
I think `cat.plot(projection="local")` looks nicer for this example.. (if you change it of course in the docstring above too)